annot-tsv.c

```diff
@@ -990,7 +990,7 @@ int main(int argc, char **argv)
             case  1 :
                 printf(
 "annot-tsv (htslib) %s\n"
-"Copyright (C) 2024 Genome Research Ltd.\n", hts_version());
+"Copyright (C) 2024-2025 Genome Research Ltd.\n", hts_version());
```

Comment:
If this PR is in place of the usual Happy New Year commit: the usual thing for these overall tool copyright notices is to just display the year of completion of the release, i.e. 2025 (see the [last paragraph here](https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html)); and the same change is needed in the `--version` output of bgzip/htsfile/tabix.